ccinput/wrapper.py

```diff
@@ -466,6 +466,9 @@ def cmd(cmd_line=None):
         for calc, outp in zip(calcs, outputs):
             with open(outp, "w") as out:
                 out.write(calc.input_file)
+            if calc.radii_parameters != '':
```

Comment:
Since `radii_parameters` is unique to `NWChemCalculation`, you must test for its existence (`hasattr(calc, 'radii_parameters)`) before accessing it, otherwise it will raise an `AttributeError` for other packages.

Also, one small tweak: when there is no output name and the input file is just displayed, the solvation radii file could be displayed as well. With `xtb`, what I've done is to simply format all the files into the `output` property. You can simply do the same and display both the input file and the auxiliary file, for completeness.

Reply:
If I understood you corretly. now everything should work